### PR TITLE
4 setup clean up repo

### PR DESCRIPTION
**Pull request by r7george**
repo cleaned up so it's in line with what's requested on the ticket

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,12 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <meta name="description" content="Web site created using create-react-app" />
     <title>React App</title>
```
Comment (Charlie-robin) [inline public/index.html:7]: Change the title from React App to OAE Digital Programme Editor?

File: public/index.html [hunk 2]
```diff
@@ -2,42 +2,12 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <meta name="description" content="Web site created using create-react-app" />
```
Comment (Charlie-robin) [inline public/index.html:6]: Add to this tag. The content should briefly describe the app.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Hey, I have made a couple of points for you to look at in the code.

You will also need to:
- delete both of the readme
- delete the yarn.lock

File: public/index.html (hunk 1 above)
Comment (r7george) [reply public/index.html]: All the requested changes have been added

File: public/index.html (hunk 2 above)
Comment (r7george) [reply public/index.html]: I've added a small description although I'm unsure how good the wording was?

File: public/index.html [hunk 3]
```diff
@@ -2,42 +2,12 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <meta name="description" content="Web site to manage the digital programs published via the digital program web-app" />
```
Comment (Charlie-robin) [inline public/index.html:6]: `content="Web application to manage the digital programs published by the OAE"`

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 3 above)
Comment (r7george) [reply public/index.html]: changes made! cheers charlie

**Pull request merged**